wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/AddParticipantActionHandler.swift

```diff
@@ -145,21 +146,55 @@ class AddParticipantActionHandler: ActionHandler<AddParticipantAction> {
             }
 
             eventProcessor.processConversationEvents([updateEvent])
-
             action.notifyResult(.success(Void()))
 
         case 204:
             action.notifyResult(.success(Void()))
-        default:
-            if response.httpStatus == 403 {
-                // Refresh user data since this operation might have failed
-                // due to a team member being removed/deleted from the team.
-                let users: [ZMUser]? = action.userIDs.existingObjects(in: context)
-                users?.filter(\.isTeamMember).forEach({ $0.refreshData() })
+
+        case 403:
+            // Refresh user data since this operation might have failed
+            // due to a team member being removed/deleted from the team.
+            let users: [ZMUser]? = action.userIDs.existingObjects(in: context)
+            users?.filter(\.isTeamMember).forEach({ $0.refreshData() })
+
+            action.notifyResult(.failure(ConversationAddParticipantsError(response: response) ?? .unknown))
```

Comment:
Suggestion for a shorter way to notify failure
```suggestion
            action.fail(with: ConversationAddParticipantsError(response: response) ?? .unknown)
```

For success it would look like

```swift
action.succeed()
```